owner/manager.go

```diff
@@ -203,12 +198,21 @@ func (m *ownerManager) campaignLoop(ctx goctx.Context, etcdSession *concurrency.
 			continue
 		}
 		m.SetOwner(true)
-
 		m.watchOwner(ctx, etcdSession, ownerKey)
 		m.SetOwner(false)
 	}
 }
 
+func (m *ownerManager) revokeSession(logPrefix string, leaseID clientv3.LeaseID) {
+	// Revoke the session lease.
+	// If revoke takes longer than the ttl, lease is expired anyway.
+	cancelCtx, cancel := goctx.WithTimeout(goctx.Background(),
+		time.Duration(ManagerSessionTTL)*time.Second)
+	_, err := m.etcdCli.Revoke(cancelCtx, leaseID)
+	cancel()
+	log.Infof("%s break campaign loop, revoke err %v", logPrefix, err)
```

Comment:
If err is nil, we do not need print it.

Reply:
The old code prints the log for breaking campaign loop, not only for the error.